packages/flutter/lib/src/material/dialog.dart

```diff
@@ -14,6 +14,7 @@ import 'dart:ui' show SemanticsRole, clampDouble, lerpDouble;
 
 import 'package:flutter/cupertino.dart';
 
+import '../services/features.dart';
```

Comment:
Discussion on the relative import on the main PR: https://github.com/flutter/flutter/pull/168437#discussion_r2093455015